docs/setup/changing-the-colors.md

```diff
@@ -208,12 +208,12 @@ The following fields must be set for each toggle:
   [9]: #accent-color
   [10]: https://github.com/squidfunk/mkdocs-material/tree/master/material/.icons
 
-### System preference
+### Inferring the initial setting from system preference
 
 [:octicons-file-code-24: Source][6] · :octicons-milestone-24: Default: _none_
 
-In order to automatically set the color palette to the user's system preference,
-a media query can be set as part of the `media` field next to the toggle
+To match a color palette to the user's system preference upon the user's first
```

Comment:
I find "to match ..." less intuitive.

Reply:
What would you suggest as an alternative?

Reply:
I kept the "To match …" for now, but word-smithed the remainder a bit. Hopefully that helps?

Reply:
No, not really.

Reply:
If my original defense was unpersuasive, what about this?

> To ~match~ _set_ a color palette _according_ to the user's system preference upon the user's first …

FWIW, I really think my original proposal is not only more concise, but more plainly and directly describes what's going on. "To set … according to …" means "match" in this context. Matching is fundamentally what's being done, no?

Reply:
Any suggestions then?

If my original defense was unpersuasive, what about this?

> To ~match~ _set_ a color palette _according_ to the user's system preference upon the user's first …

FWIW, I really think my original proposal is not only more concise, but more plainly and directly describes what's going on. "To set … according to …" means "match" in this context. Matching is fundamentally what's being done, no?